class.application-passwords-list-table.php

```diff
@@ -71,7 +71,7 @@ protected function display_tablenav( $which ) {
 	}
 
 	public function single_row( $item ) {
-		echo '<tr data-slug="' . Application_Passwords::password_unique_slug( $item ) . '">';
+		echo '<tr data-slug="' . esc_attr( Application_Passwords::password_unique_slug( $item ) ) . '">';
```

Comment:
Good catch.  I don't know if it will ever be useful in practice, as the content will only ever be a substr of a md5 hash, but it's still a good practice to be in.